6.2.строковый_тип_данных/6.2.13.py

```diff
@@ -0,0 +1,7 @@
+line = input()
+if "суббота" in line:
+    print("YES")
+elif "воскресенье" in line:
+    print("YES")
```

Comment:
Один `elif` можно убрать:
```suggestion
if "суббота" in string or "воскресенье" in string:
    print("YES")
```